src/Features/Core/Portable/MetadataAsSource/AbstractMetadataAsSourceService.WrappedNamedTypeSymbol.cs

```diff
@@ -146,6 +146,8 @@ public ImmutableArray<SymbolDisplayPart> ToMinimalDisplayParts(SemanticModel sem
 
         public bool IsExtension => _symbol.IsExtension;
 
+        public IParameterSymbol ExtensionParameter => _symbol.ExtensionParameter;
```

Comment:
>_symbol.ExtensionParameter [](http://example.com/codeflow?start=54&length=26)

This looks suspicious, a wrapper should probably have its own extension parameter symbol (perhaps a wrapper itself).